Part 1/Chapter 4 - Kleisli Categories/rgs_solutions.apln

```diff
@@ -0,0 +1,25 @@
+:Namespace rgs_solutions
+
+    nullopt ← 0 ⍬
+    Val ← ⊃⌽
+    HasVal ← ⊃
+    MkOpt ← 1,⊂
+    
+    _KC_ ← {
```

Comment:
@codereport regarding the "APL cannot return functions", it might be interesting to note that you can write operators and then use them to _derive_ new functions. This `_KC_` operator here does *not* return a new function, but does allow me to compose arbitrarily many safe functions.